src/Adapter/Product/AdminProductDataProvider.php

```diff
@@ -96,7 +96,7 @@ public function getPersistedFilterParameters()
                 'action' => 'catalogAction',
             ]);
 
-            /** @var $filter AdminFilter */
+            /** @var AdminFilter|null $filter */
```

Comment:
Maybe put it just above the variable assignation?

Reply:
Fixed